internal/tiger/cmd/db.go

```diff
@@ -182,14 +198,25 @@ Examples:
 				return fmt.Errorf("connection pooler not available for this service")
 			}
 
-			// Launch psql with additional flags
-			return launchPsqlWithConnectionString(details.String(), psqlPath, psqlFlags, service, dbConnectRole, cmd)
+			cmd.SilenceUsage = true
```

Comment:
I believe that `getServiceDetails(cmd, serviceArgs)`, which is called above, already sets `cmd.SilenceUsage = true` (see [here](https://github.com/timescale/tiger-cli/blob/b936b4a160a1914f50feb9b2ff96179f942a694d/internal/tiger/cmd/db.go#L792)). So I think this line is unnecessary/duplicative.